.yamato/com.unity.ml-agents-coverage.yml

```diff
@@ -33,7 +33,7 @@ test_coverage_{{ package.name }}_{{ platform.name }}_{{ editor.version }}:
             {% if package.name == "com.unity.ml-agents.extensions" %}
             pull_request.changes.any match "com.unity.ml-agents.extensions/**" OR
             {% endif %}
-            pull_request.changes.any match ".yamato/com.unity.ml-agents-test.yml")
+            pull_request.changes.any match ".yamato/com.unity.ml-agents-coverage.yml")
```

Comment:
This was triggering even though it wasn't supposed to (now it's triggering because it was modified).

Reply:
thanks